### PR TITLE
Readme update 

### LINKED ISSUE #32: update README.md
clone the project and install as per instructions then made following changes to file:
- update section **Command line options**.
- update all the **Examples** as per new options.
- update **how to install** section.

### LINKED ISSUE #44: Add new features
Feature:
1. User should be able to open url in browser through command line option.

### PR DESCRIPTION
**Pull request by ganeshhubale**
Fix  #32 #29 
- README.md content updated
    - Feature
    - Examples
    - Command line options
    - Dependencies

### CODE REVIEW

**Comment by ganeshhubale** [comment]
#46 Fixed.
Updated code with respect to improve readability.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,20 @@
-# Readit - Command line tool
+# Readit - Command Line Tool
 
 
 ## Introduction
-Readit is command line bookmark manager. It is a command line utility to add, delete, update the bookmarks. It is a powerful bookmark manager written in Python3 and SQLite3.
+Readit is command line bookmark manager. It is a command line utility to add, delete, update the bookmarks. It is a powerful bookmark manager written in Python and SQLite3.
```
Comment (bhavin192) [inline README.md]: How about writing this as,
It is a powerful bookmark manager written in Python. It uses SQLite3 to store the bookmarks.

Comment (ganeshhubale) [reply README.md]: @bhavin192 
Yes, It will be fine too. Also, I have introduced those utilities in features.

Comment (bhavin192) [reply README.md]: I was talking about the last sentence only not the sentence containing small list of features. It is good to mention those features in first few lines.

Comment (ganeshhubale) [reply README.md]: Yes, I understood. I will update introduction.

**Pull request merged**